assets/src/application/ui/input/Button/ButtonGroup/style.scss

```diff
@@ -1,20 +1,21 @@
 .ee-btn-group {
-	.ee-btn {
-		margin-right: 0;
+	margin-right: var(--ee-margin-default);
 
-		&:first-child {
+	.ee-btn.ee-btn {
+		margin: 0;
+
+		&--no-border-radius {
+			border-radius: 0;
+		}
```

Comment:
no need for an explicit class

```suggestion
		&:not(:first-child),
		&:not(:last-child) {
			border-radius: 0;
		}
```

Reply:
great suggestion, applied, thx

Reply:
that was applying the property to all buttons:

![image](https://user-images.githubusercontent.com/1477453/83056814-b27b0c80-a05e-11ea-9f9c-d1a1f110c5af.png)

Reply:
applied a slightly different selector: 

![image](https://user-images.githubusercontent.com/1477453/83056907-d76f7f80-a05e-11ea-8176-fed6453255c5.png)